example/IO/cmesh/vtk/t8_cmesh_read_from_vtk.cxx

```diff
@@ -162,11 +162,13 @@ main (int argc, char **argv)
                          "The prefix of the output-file.");
   sc_options_add_int (opt, 'c', "num_cell_values", &num_keys, 0,
                       "Number of values per cell stored in the vtk-file.");
-  sc_options_add_int (opt, 't', "type_of_file", &vtk_file_type, -1,
-                      " Set the type of the data in the file.\n"
-                      "0 for vtkUnstructuredGrid \n" "1 for vtkPolyData");
   sc_options_add_bool (opt, 'p', "partition", &partition, 0,
-                       "If set, partition the cmesh uniformly.\n");
+                       "If set, partition the cmesh uniformly.");
+  sc_options_add_int (opt, 't', "type_of_file", &vtk_file_type, -1,
```

Comment:
Because the filetype does not uniquely determine the Datatype stored in it. It is possible to save vtkPolyData in a vtk-legacy file. In that case you would set the option to use the polydata-reader for a vtk-file. If you want to, we can ignore these edge-cases in the example and detect the filetype automatically.